app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -62,7 +47,16 @@ const Notes = () => {
       />
       {notes.length ? (
         <Scrollable className="neeto-ui-bg-gray-100 w-full space-y-6 p-6">
-          <Card created_at={new Date()} />
+          {notes.map(note => (
+            <Card
+              assignedContactId={note.assignedContactId}
+              created_at={note.created_at}
+              description={note.description}
+              key={`${note.title}1`}
```

Comment:
I don't understand your reasoning for keeping the title as the key. How can you assume that titles will be unique? Please use a different key. You may use [uuid](https://www.npmjs.com/package/uuid) package if necessary.

Reply:
@LiyakathMohammed This is actually a really careless mistake. It shows that you are not thinking about all scenarios properly. Please ensure that you pay very close attention to the code that you are writing and consider the different scenarios it has to deal with as part of your self-review!